### ISSUE TITLE
`EXIT` Command Exception in RCON

### ISSUE THREAD

**Issue by D1G1T4L3CH0** [DeathCradle/Terraria-s-Dedicated-Server-Mod#156]
Probably shouldn't even allow it, only allow /restart maybe.

```
 Issue parsing remote command from blah:
TDSM.API.Misc.ExitException: blah@CONSOLE requested that TDSM is to shutdown.
  at TDSM.Core.Entry.Exit (ISender sender, TDSM.API.Command.ArgumentList args) [0x00000] in <filename unknown>:0
  at TDSM.API.Command.CommandInfo.Run (ISender sender, TDSM.API.Command.ArgumentList args) [0x00000] in <filename unknown>:0
  at TDSM.API.Command.CommandParser.ParseAndProcess (ISender sender, System.String line) [0x00000] in <filename unknown>:0
```

**Comment by DeathCradle** [comment]
Fixed in: 06b31e5521d33527014658b2372be3cdb033865e

**Comment by D1G1T4L3CH0** [comment]
Nearly fixed...
```
31/07/2015 8:11:29 PM RCon> User "d1g1t4l3ch0" has logged in to a remote console from blah:56486
31/07/2015 8:11:32 PM RCon> Remote command from d1g1t4l3ch0 (blah:56486): "list"
31/07/2015 8:11:34 PM RCon> Remote command from d1g1t4l3ch0 (blah:56486): "exit"
31/07/2015 8:11:34 PM RCon> Exiting on request.

Saving world data: 100%
Validating world save: 97%
Backing up world file
31/07/2015 8:11:36 PM RCon> Issue parsing remote command from d1g1t4l3ch0@blah:56486:
TDSM.API.Misc.ExitException: d1g1t4l3ch0@CONSOLE requested that TDSM is to shutdown.
  at TDSM.Core.Entry.Exit (ISender sender, TDSM.API.Command.ArgumentList args) [0x00000] in <filename unknown>:0
  at TDSM.API.Command.CommandInfo.Run (ISender sender, TDSM.API.Command.ArgumentList args) [0x00000] in <filename unknown>:0
  at TDSM.API.Command.CommandParser.ParseAndProcess (ISender sender, System.String line) [0x00000] in <filename unknown>:0
31/07/2015 8:11:36 PM Run> TDSM Core Module disabled.
```
